environment/src/main/resources/schema/app/20210525113034_CB-12118_persistence_for_encryptionKeyResourceGroupName_on_Azure.sql

```diff
@@ -0,0 +1,11 @@
+-- // CB-12118 persistence for encryptionKeyResourceGroupName on Azure
+-- Migration SQL that makes the change goes here.
+ALTER TABLE environment_parameters DROP COLUMN IF EXISTS encryption_keyUrl;
+ALTER TABLE environment_parameters ADD COLUMN IF NOT EXISTS encryption_key_resource_group_name text;
```

Comment:
`resource_group_name` is a `VARCHAR(255)`. What is the reasoning behind using `TEXT` here? In any case, `TEXT` would need `@Column(columnDefinition = "TEXT")` in `AzureParameters`.